ember-toucan-core/src/components/form/input-field.hbs

```diff
@@ -0,0 +1,27 @@
+<div
+  class="flex flex-col {{if @isDisabled 'text-disabled cursor-not-allowed'}}"
```

Comment:
Is the `cursor-not-allowed` class something we want here?

Reply:
Will remove it for now...I was thinking we wanted to whole input field to be disabled but probably better to adjust this (if even needed!) when design is more finalized.

BTW (not sure if this is clear or not) `cursor-not-allowed` changes the cursor style, it doesn't hide it. https://tailwindcss.com/docs/cursor

Unlike what the naming suggests...